### PR TITLE
Fixes #18761: Add solaris to 6.2 release data

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/18761

### CODE REVIEW

**Review by VinceMacBuche — APPROVED** [review]

File: release-data/rudder-6.1.cson [hunk 1]
```diff
@@ -13,7 +13,7 @@ lts = false
 
 # Supported architecture for each distribution
 architectures = { 
-                  "armbian-8":      [ "armhf", "arm64" ],
+                  "armbian-8":      [ "armhf" ],
```
Comment (VinceMacBuche) [inline release-data/rudder-6.1.cson:16]: WHy keeping armhf here  and not in https://github.com/Normation/rudder-tools/pull/651/files#diff-c4e06da963cd3b736367273f7c9c7615f9688801abeca66117b3b0fea7698ca2L16 ?

**Review by VinceMacBuche — CHANGES_REQUESTED** [review]

File: release-data/rudder-6.0.cson [hunk 2]
```diff
@@ -16,7 +16,7 @@ lts = true
 
 # Supported architecture for each distribution
 architectures = { 
-                  "armbian-8":      [ "armhf", "arm64" ],
+                  "armbian-8":      [ "armhf" ],
```
Comment (VinceMacBuche) [inline release-data/rudder-6.0.cson:19]: Same here

File: release-data/rudder-6.1.cson (hunk 1 above)
Comment (peckpeck) [reply release-data/rudder-6.1.cson:16]: because armhf was built on 6.0 and 6.1

Comment (VinceMacBuche) [reply release-data/rudder-6.1.cson:16]: was built, but is not anymore ?

Comment (peckpeck) [reply release-data/rudder-6.1.cson:16]: not anymore on 6.0 since most 6.0 tasks have been disabled

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**